frontend/src/interfaces/ltsaModels.ts

```diff
@@ -2041,13 +2041,13 @@ export interface TitleOwner {
    * @type {string}
    * @memberof TitleOwner
    */
-  surnameOrCorpName1: string;
```

Comment:
this is surprising, these are generated so I wouldn't expect any of these to be wrong. Are you sure about this?